ush/forecast_postdet.sh

```diff
@@ -212,7 +217,11 @@ EOF
         if [[ "${DO_JEDIATMVAR:-NO}" == "YES" ]]; then
           increment_file="${COMIN_ATMOS_ANALYSIS}/${RUN}.t${cyc}z.cubed_sphere_grid_${PREFIX_ATMINC}${inc_file}"
         else
-          increment_file="${COMIN_ATMOS_ANALYSIS}/${RUN}.t${cyc}z.${PREFIX_ATMINC}${inc_file}"
+          if [[ "${RUN}" == "gcafs" ]]; then
+            increment_file="${COMIN_ATMOS_OFFLINE_ANALYSIS}/gcdas.t${cyc}z.${PREFIX_ATMINC}${inc_file}"
```

Comment:
Is there a reason why `COMIN_ATMOS_ANALYSIS` cannot mean `COMIN_ATMOS_OFFLINE_ANALYSIS` when `RUN=gcafs`?  In other words, why make up a variable `COMIN_ATMOS_OFFLINE_ANALYSIS`?

Reply:
Mainly only that it makes the J-Job definition of COMIN_ATMOS_ANALYSIS more complicated. I'm happy to put the if/else there instead of adding this if you'd prefer that

Reply:
See commit https://github.com/NOAA-EMC/global-workflow/pull/3745/commits/c2b2dbe8efdc72b589da2921f16e79d0a3d6ef46 for this change, if this is what you would prefer